code/game/machinery/computer/labor.dm

```diff
@@ -0,0 +1,165 @@
+var/list/labor_console_categories = list(
+	"Command" = command_positions,
+	"Civilian" = civilian_positions,
+	"Security" = security_positions,
+	"Engineering" = engineering_positions,
+	"Medical" = medical_positions,
+	"Science" = science_positions,
+	"Cargo" = cargo_positions,
+	)
+
+/obj/machinery/computer/labor
+	name = "Labor Administration Console"
+	desc = "According to the manual, you need to take a six-week Labor Administration Associate Training Course before you're qualified to navigate this console's complex interface. Being a HoP is hard work."
+	icon = 'icons/obj/computer.dmi'
+	icon_state = "comm_logs"
+	light_color = LIGHT_COLOR_GREEN
+	req_access = list(access_hop)
+	circuit = "/obj/item/weapon/circuitboard/labor"
+
+	var/freeing = "" //If this variable is set with a job's title, the user will be prompted to swipe to free up a job slot.
+	var/toggling_priority = "" //If this variable is set with a job's title, the user will be prompted to swipe to prioritize/deprioritize.
+	var/selected_category = "Civilian"
+	var/list/swipe_sounds = list('sound/effects/cardswipe1.ogg', 'sound/effects/cardswipe2.ogg', 'sound/effects/cardswipe3.ogg')
+
+/obj/machinery/computer/labor/New()
+	..()
+	job_master.labor_consoles += src
+
+/obj/machinery/computer/labor/Destroy()
+	job_master.labor_consoles -= src
+	..()
+
+/obj/machinery/computer/labor/attack_ai(var/mob/user as mob)
+	add_hiddenprint(user)
+	return attack_hand(user)
+
+/obj/machinery/computer/labor/attack_paw(var/mob/user as mob)
```

Comment:
Could use feedback explaining why you can't interact. Even better if it did a dexterity check and if passed goes to attack_hand, for methylin fueled monkey HOP

Reply:
Turns out this was entirely redundant and just removing it gives feedback